### DISCUSSION TITLE
Test and Build stage in CI Pipeline

### DISCUSSION THREAD

**Discussion by KrisAff84** [LUIT-POD1-HealthCare/HealthCare-Website#27] [Ideas]
Hey @jo016wh and @KayKaks, I just pushed my branch pipeline_dev. I think I'm on the right track with the build and test stages, but take a look. Haven't made a PR because I'm still not exactly sure about a few things. Plus, we need to input a CodeBuild service role arn, which is probably in the sandbox account, or we'd have to ask Troy to create it. Still working out some kinks, but I think I'm getting closer.

I went ahead and just started working on that CodeBuild step, because it kind of involved the same things I had to do for unit tests, just involving slightly different commands, so it made sense to just work on it together. If you've been researching it too, let me know if you think it's on the right track. 

Basically, I have two CodeBuild steps in the main.tf file now. The first one performs the unit tests, and then the second one extracts the index.html so that only that file goes to the bucket for deployment. Sorry if I stepped on some toes, just kind of had a Eureka moment and it just made sense to start writing this stuff down.
